feature-libs/product-configurator/rulebased/components/attribute/footer/configurator-attribute-footer.module.ts

```diff
@@ -13,6 +14,15 @@ import { ConfiguratorAttributeFooterComponent } from './configurator-attribute-f
     I18nModule,
     IconModule,
   ],
+  providers: [
+    provideDefaultConfig(<ConfiguratorAttributeCompositionConfig>{
+      productConfigurator: {
+        attributeComponentAssignment: {
```

Comment:
Nitpick: for ergonomics of using this config, did you consider shorter config key instead of `attributeComponentAssignment`? maybe simply `attributeComponent`? or just `attribute`?